LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
* [ ] FYI The Example App also mentions the license in the package.json (https://github.com/Scrivito/scrivito_example_app_js/blob/master/package.json#L4)